src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier SUPPLIER = new ColorSupplier();
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
```

Comment:
The method signature has an unnecessary empty line above it. Please remove the empty line to comply with the code style.